pytket/docs/conf.py

```diff
@@ -57,6 +57,7 @@
     "sphinx.ext.autosummary",
     "sphinx.ext.intersphinx",
     "sphinx.ext.mathjax",
+    "sphinx_copybutton",
```

Comment:
Does the copied text exclude the initial `>>> ` in the cells? (If it doesn't then it's less useful for copying commands...)

Reply:
Yes it does copy the `>>>` in the cells. I thought this could also be nice for pip installation commands etc. Should I keep the copy button in?

Reply:
OK, still useful in some cases, so may as well keep it.